src/__swaps__/screens/Swap/components/SwapOutputAsset.tsx

```diff
@@ -41,14 +43,38 @@ function SwapOutputActionButton() {
 }
 
 function SwapOutputAmount() {
-  const { focusedInput, SwapTextStyles, SwapInputController, AnimatedSwapStyles } = useSwapContext();
+  const { navigate } = useNavigation();
+  const { focusedInput, SwapTextStyles, SwapInputController, AnimatedSwapStyles, outputQuotesAreDisabled } = useSwapContext();
+
+  const handleTapWhileDisabled = useCallback(() => {
+    const inputTokenSymbol = useSwapsStore.getState().inputAsset?.symbol;
```

Comment:
You can probably combine the `getState()` calls:
```
const {
    inputAsset,
    outputAsset,
  } = useSwapsStore.getState();
```

Or alternatively use the shared values since it'll be able to read the latest values at the time this is called.